pkg/cmd/grafana-server/main.go

```diff
@@ -51,10 +51,11 @@ func (e exitWithCode) Error() string {
 
 func main() {
 	var (
-		configFile = flag.String("config", "", "path to config file")
-		homePath   = flag.String("homepath", "", "path to grafana install/home path, defaults to working directory")
-		pidFile    = flag.String("pidfile", "", "path to pid file")
-		packaging  = flag.String("packaging", "unknown", "describes the way Grafana was installed")
+		configFile  = flag.String("config", "", "path to config file")
+		homePath    = flag.String("homepath", "", "path to grafana install/home path, defaults to working directory")
+		pidFile     = flag.String("pidfile", "", "path to pid file")
+		packaging   = flag.String("packaging", "unknown", "describes the way Grafana was installed")
+		distributor = flag.String("distributor", "unknown", "name of the distributor of the package")
```

Comment:
why is this a command line option, would have thought it be an ini setting (that way it can be set using env variable as well)

Reply:
Yes, it makes sense to put this under the `analytics` settings in the ini.

Reply:
yes that was my thinking as well

Reply:
Agree. I'm not sure what I was thinking... I'll fix this once the kids are asleep. Hopefully we can include it in 7.4.1

Reply:
:fist_raised: force pushed!